source/frontend/src/features/acquisition/list/AcquisitionFilter/AcquisitionFilter.tsx

```diff
@@ -1,31 +1,28 @@
 import { Formik, FormikHelpers, FormikProps } from 'formik';
-import React from 'react';
+import React, { useEffect, useMemo } from 'react';
 import { Col, Row } from 'react-bootstrap';
 import styled from 'styled-components';
 
 import { ResetButton, SearchButton } from '@/components/common/buttons';
-import { Form, Input, Select } from '@/components/common/form';
+import { Form, Input, Multiselect, Select } from '@/components/common/form';
 import { SelectInput } from '@/components/common/List/SelectInput';
 import { ACQUISITION_FILE_STATUS_TYPES } from '@/constants/API';
+import { useAcquisitionProvider } from '@/hooks/repositories/useAcquisitionProvider';
 import useLookupCodeHelpers from '@/hooks/useLookupCodeHelpers';
 import { mapLookupCode } from '@/utils';
+import { formatApiPersonNames } from '@/utils/personUtils';
 
-import { IAcquisitionFilter } from '../interfaces';
+import { AcquisitionFilterModel, Api_AcquisitionFilter } from '../interfaces';
 
-export interface IAcquisitionFilterProps {
-  filter?: IAcquisitionFilter;
-  setFilter: (filter: IAcquisitionFilter) => void;
+interface MultiSelectOption {
+  id: string;
+  text: string;
 }
```

Comment:
nit: This interface is exported in `source/frontend/src/features/acquisition/list/interfaces.ts` - perhaps we should import it from there instead of repeating here?

Reply:
good idea